application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -405,4 +411,17 @@ public FeatureBlacklistConfig getFeatureBlacklistConfig() {
     public String getSelectRolesChannelPattern() {
         return selectRolesChannelPattern;
     }
+
+    /**
+     * Gets the categories by this name to showcase the total member count of the server.
+     *
+     * @return the categories name types
```

Comment:
i find this javadoc confusing wording-wise.

instead:

Gets the pattern matching the category that is used to display the total member count.

Reply:
not changed

Reply:
the `return` part still has confusing naming. perhaps `the pattern matching the category`